components/ui/AppMosaic.vue

```diff
@@ -11,13 +11,13 @@
           class="app-mosaic__element-copy"
           :class="`app-mosaic__element-copy_${position}`"
         >
-          <dt class="h3" v-text="title" />
+          <dt class="h3" v-text="title"></dt>
           <!-- eslint-disable vue/no-v-html -->
           <dd
             class="app-mosaic__element-copy-description"
             :class="`app-mosaic__element-copy-description_${position}`"
             v-html="description"
-          />
+          ></dd>
```

Comment:
Why are we changing these?

Reply:
Cleaned up in https://github.com/Qiskit/qiskit.org/pull/3017/commits/4182ab1ac9a79512c57a38396351d12a33df66b6